gatsby-config.js

```diff
@@ -20,6 +26,15 @@ module.exports = {
     'gatsby-plugin-eslint',
     'gatsby-plugin-remove-trailing-slashes',
     'gatsby-plugin-netlify',
+    {
```

Comment:
I changed GATSBY_ALGOLIA_ADMIN_KEY to ALGOLIA_ADMIN_KEY. Variables prefixed with "GATSBY_" are automatically placed in a global variable on the client, and we don't want to share it.

Reply:
OK, knew that CRA worked this way too. Will change it

Reply:
@kevee Changed to `process.env.ALGOLIA_ADMIN_KEY`. Local build is passing.